openreview/matching.py

```diff
@@ -15,35 +14,49 @@ def get_conflicts(author_profiles, user_profile):
         authors_domain_conflicts.update(author_domain_conflicts)
         authors_relation_conflicts.update(author_relation_conflicts)
 
+    user_domain_conflicts, user_relation_conflicts = get_profile_conflicts(user_profile)
 
-	user_domain_conflicts, user_relation_conflicts = get_profile_conflicts(user_profile)
+    conflicts.update(authors_domain_conflicts.intersection(user_domain_conflicts))
+    conflicts.update(authors_relation_conflicts.intersection(user_relation_conflicts))
 
-	conflicts.update(authors_domain_conflicts.intersection(user_domain_conflicts))
-	conflicts.update(authors_relation_conflicts.intersection(user_relation_conflicts))
-
-	return list(conflicts)
+    return list(conflicts)
 
 def get_profile_conflicts(profile):
     domain_conflicts = set()
     relation_conflicts = set()
 
     profile_domains = []
     for e in profile.content['emails']:
-    	profile_domains += tools.subdomains(e)
+        profile_domains += get_domains(e, subdomains = True)
     domain_conflicts.update(profile_domains)
 
     institution_domains = []
     for h in profile.content.get('history', []):
-    	domain = h.get('institution', {}).get('domain', None)
-    	if domain:
-    		institution_domains += tools.subdomains(domain)
+        domain = h.get('institution', {}).get('domain', None)
+        if domain:
+            institution_domains += get_domains(domain, subdomains = True)
     domain_conflicts.update(institution_domains)
 
     domain_conflicts.update(institution_domains)
 
-    relation_conflicts.update([r['email'] for r in profile.content.get('relations', [])])
+    relation_conflicts.update([get_domains(r['email'], subdomains = False) for r in profile.content.get('relations', [])])
```

Comment:
I'm not sure if we need to compute the relation conflicts matching the domains or the full email address. I realized that is the last one, in this case, I will change the backend so we can get the email addresses not masked for the superuser.